container-search/src/main/java/com/yahoo/search/statistics/ElapsedTime.java

```diff
@@ -96,7 +96,8 @@ public long fillTime() {
     public long totalTime() {
         long total = 0L;
         for (TimeTracker track : tracks) {
-            total  += track.totalTime();
+            if (track.first() != 0L)
```

Comment:
I do not think this is needed here. However, it would be useful with such a guard in the 'first' function (similar to the 'firstFilled' function). The 'first' function will ask all trackers about the first entry and if there is an empty tracker it will shadow all entries (including the actual first one) from the other trackers.